backend/Techfesia2019/accouts/models.py

```diff
@@ -1,3 +1,25 @@
 from django.db import models
+from django.contrib.auth.models import AbstractUser
+from django.utils.translation import ugettext_lazy as _
+from django.conf import settings
 
-# Create your models here.
+
+class User(AbstractUser):
+    username = models.CharField(max_length=12, unique=True)
+    email = models.EmailField(_('email address'), unique=True)
+    phone_number = models.CharField(max_length=12, null=False, blank=False, unique=True)
+    birth_date = models.DateField()
+    college_name = models.CharField(null=False,max_length=40)
+
+    USERNAME_FIELD = 'email'
```

Comment:
username field should be username only and not email.
django-rest-framework causes some problems if we change username field